<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wIntermedia</class>
 <widget class="QMainWindow" name="wIntermedia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>398</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>398</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>398</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ACOES - Subsistemas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bFinanciero">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>304</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>304</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>304</width>
      <height>48</height>
     </size>
    </property>
    <property name="text">
     <string>Subsistema Financiero</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bUsuarios">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>304</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>304</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>304</width>
      <height>48</height>
     </size>
    </property>
    <property name="text">
     <string>Subsistema de Usuarios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bApadrinamiento">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>304</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>304</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>304</width>
      <height>48</height>
     </size>
    </property>
    <property name="text">
     <string>Subsistema de Apadrinamiento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bPerfil">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>53</width>
      <height>46</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>53</width>
      <height>46</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>53</width>
      <height>46</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bAtras">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>63</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>63</width>
      <height>27</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>63</width>
      <height>27</height>
     </size>
    </property>
    <property name="text">
     <string>&lt; Atr√°s</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
